src/components/NavigationBar/NavigationBar.js

```diff
@@ -0,0 +1,55 @@
+import React, { useRef } from 'react'
+import { connect } from 'react-redux';
+import { Navbar, Nav, Form, FormControl } from 'react-bootstrap';
+import ToastPokemon from '../ToastPokemon'
+import navigationStyle from './NavigationBar.module.css'
+import { storePokemonSearching } from '../../redux/actions/navigationBarActions'
+
+const NavigationBar = ({ storePokemonSearching, isFetchingSearch }) => {
+  const input = useRef()
+
+  const getInput = (inputEvent) => {
```

Comment:
I recommend being more descriptive with the name of the functions, getInput would describe that this function gets a input tag from the template, remember to name the functions for what they do, not where they are used